python/ray/experimental/sgd/pytorch/distributed_pytorch_runner.py

```diff
@@ -0,0 +1,129 @@
+import logging
+import os
+import torch.distributed as dist
+import torch.utils.data
+
+from ray.experimental.sgd.pytorch.pytorch_runner import PyTorchRunner
+
+logger = logging.getLogger(__name__)
+
+
+class DistributedPyTorchRunner(PyTorchRunner):
+    """Manages a distributed PyTorch model replica"""
+
+    def __init__(self,
+                 model_creator,
+                 data_creator,
+                 optimizer_creator,
+                 config=None,
+                 batch_size=16,
+                 backend="gloo"):
+        """Initializes the runner.
+
+        Args:
+            model_creator (dict -> torch.nn.Module): creates the model using
```

Comment:
maybe avoid duplicating docstring and just instead say "see xx.py"

Reply:
The args aren't quite the same since `DistributedPyTorchRunner` takes a `backend` argument.

Reply:
I meant individual arg docstrings (model creator: see .)

Reply:
Got it, fixed!